docs/index.asciidoc

```diff
@@ -89,6 +89,7 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Setting |Input type|Required
+| <<plugins-{type}s-{plugin}-ecs_compatibility>> | <<string,string>>|No
```

Comment:
Doc: Please order alphabetically in both the table and doc description.  Otherwise, build cleanly and LGTM>